src/main/kotlin/ui/fragment/IssueListCellFragment.kt

```diff
@@ -117,7 +145,6 @@ class IssueListCellFragment : SuspendingListCellFragment<Issue>() {
 
             timeSummaryProperty.set("$timeSummaryFirstHalf of estimated ${updatedIssue.timeEstimate}")
             progressPercentageProperty.set(percentComplete)
-            shouldNotShowEstimateProperty.set(updatedIssue.timeSpent.totalMinutes == 0L)
```

Comment:
Move the `timeSummaryProperty` and `progressPercentageProperty` stuff into bindings, but keep the class applications in this function